argument order for filter and threshold changed in find_photons()

ambarb opened the pull request:
See https://github.com/NSLS-II/centroids/issues/24

just switched `250` and `3` in the call to `find_photons()`

Left output in place as this is how it was rendered before this PR

ambarb committed:
argument order for filter and threshold changed in find_photons()

ambarb commented:
@stuwilkins or @stuartcampbell  i see that you two are the ones associated with this repo.  I cannot ask for your explicit review on the right-hand side at the top.

I don't know how to fix the things that failed (or which ones can be safely ignored).  

We ran the notebook and code using python3.9.  We used scipy environment on jupyterhub so the available centroids is:
![image](https://github.com/NSLS-II/centroids/assets/13630233/d8d1fc2f-d821-46d2-aa43-13d6daf2cbc9)

mrakitin commented:
@ambarb, I've created a tag v0.2.0rc3, this change can go into the next release candidate. How ready is your PR?
My plan is to rework the CI for this repository to use GitHub Actions rather than Azure DevOps - that way we can have better CI runs.

ambarb commented:
@mrakitin no.  there is a mistake here and I need to look back at my slack conversation with the summer student that spend the summer looking at the c-code.  

@padraic-shafer FYI given your question on slack

ambarb committed:
not an argument order issue.  an additional argument seems to have been added to find_photons()

ambarb commented:
@mrakitin i fixed the notebook that had the wrong argument

for this branch, the version is:
```python
pycentroids._pycentroids.__version__
```
`'67bb8de+'`

do i need to do anything to get the correct tag?  The notebook example that is in production shows tag associated with a version release.  [see the last cell](https://github.com/NSLS-II/centroids/blob/master/examples/Single_Photon_Test.ipynb). mine is just a generic commit 

as for the results, added `None` for what I am assuming is the new `filter` argment finds the same number of of possible events, but some of the finer details make it appear there are some improvements in reducing more outliers.
